mobile/examples/object_detection/ios/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
Do we need this?

Reply:
yes, this is necessary.:D

Reply:
hm, I managed to have a Podfile without it. could be that it took awhile for the pod to go to the CDN after it was initially released and this was briefly necessary

Reply:
oh okay I see. will update later then.